lab-goat/__test_/read-write-bitmap.test.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const originalObj = require('../lib/read-write-bitmap');
+
+
+describe('test to see if greyscale works', () => {
+  test('test image should equal greyscale image', () => {
+    originalObj.greyOut();
+    expect(`${__dirname}/../../assets/test-bitmap.bmp`).toEqual(`${__dirname}/../../assets/greyscale-bitmap.bmp`);
+  });
+  test('test image should equal white out image', () => {
```

Comment:
I'm guessing that a deep comparison of these two buffers works, but confirm for me? It's a pretty abstract way to test, which doesn't leave too much in the way of dynamic test content.